askama/src/error.rs

```diff
@@ -71,6 +72,13 @@ impl From<::serde_json::Error> for Error {
     }
 }
 
+impl From<Infallible> for Error {
```

Comment:
Every single commit makes the return type `Result<_, Infallible>` for its respective filter. I could merge the change with the first commit of the PR, but I guess it can keep it its own change.